scripts/deployment/pip/requirements/2_rsr.txt

```diff
@@ -50,7 +50,7 @@ django-embed-video==0.11
 
 # Database requirements
 South==0.8.4
-psycopg2==2.6.2
+psycopg2==2.7.4
```

Comment:
Travis not happy anyway 😢

Reply:
Travis actually built it successfully. :smile:

Reply:
Missed that 😛